@here/harp-webpack-utils/scripts/HarpWebpackConfig.js

```diff
@@ -35,18 +51,20 @@ export function addHarpWebpackConfig(config?: Configuration, harpConfig?: HarpWe
         },
         mode
     };
-    const typescriptConfig: Configuration = {
+    const typescriptConfig /*: Configuration*/ = {
         resolve: {
             extensions: [".web.ts", ".ts", ".tsx"]
```

Comment:
> `tsx`

Again, harp.gl doesn't require `tsx` anywhere.
(Some of our apps, use it indded, but it's out of scope of this tool).